xtest/runner.py

```diff
@@ -133,14 +170,14 @@ def gen_files(serial):
     return ct, rt, mf
 
 
-def encrypt(sdk, pt_file, ct_file, mime_type="application/octet-stream"):
-    c = [sdk, "encrypt", pt_file, ct_file, "--mimeType", mime_type]
+def encrypt(sdk, pt_file, ct_file, mime_type="application/octet-stream", nano=False):
+    c = [sdk, "encrypt", pt_file, ct_file, str(nano), "--mimeType", mime_type]
     logger.info("Invoking subprocess: %s", " ".join(c))
     subprocess.check_call(c)
 
 
-def decrypt(sdk, ct_file, rt_file):
-    c = [sdk, "decrypt", ct_file, rt_file]
+def decrypt(sdk, ct_file, rt_file, nano=False):
+    c = [sdk, "decrypt", ct_file, rt_file, str(nano),]
```

Comment:
ugh I hope someday the clients all support decyrpt without having to pass in the file type but see above for the note on changing the parameter to be `nano` instead of `true`.

```suggestion
    c = [sdk, "decrypt", ct_file, rt_file, "nano" if nano else "ztdf",]
```